litellm/model_prices_and_context_window_backup.json

```diff
@@ -1144,6 +1144,26 @@
         "mode": "embedding",
         "source": "https://cloud.google.com/vertex-ai/generative-ai/docs/learn/models#foundation_models"
     },
+    "text-embedding-preview-0409": {
+        "max_tokens": 3072,
+        "max_input_tokens": 3072,
+        "output_vector_size": 768,
+        "input_cost_per_token": 0.00000000625,
+        "output_cost_per_token": 0,
+        "litellm_provider": "vertex_ai-embedding-models",
+        "mode": "embedding",
+        "source": "https://cloud.google.com/vertex-ai/generative-ai/docs/learn/models#foundation_models"
```

Comment:
@Dev-Khant the source is incorrect here - i don't see `text-embedding-preview-0409` on https://cloud.google.com/vertex-ai/generative-ai/docs/learn/models#foundation_models

Can you please link the source you got the pricing information from 


Also - is the output vector size only 768? On the example here I see 256 - https://cloud.google.com/blog/products/ai-machine-learning/google-cloud-announces-new-text-embedding-models

Reply:
hey @demux79 thanks - do you have a link to the pricing for this too?

Reply:
ok - merging. will fix this on our end.